### PR TITLE
Mitheysh.asokan/x ray billing updates

### PR DESCRIPTION
**Pull request by mitheysh-asokan**
### What does this PR do?

Changes made to the aws_lambda and aws_xray READMEs to better reflect the billing implications of enabling the AWS X-Ray integration. 

### Preview link

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### CODE REVIEW

File: content/en/account_management/billing/apm_distributed_tracing.md [hunk 1]
```diff
@@ -74,15 +74,26 @@ Agent running on 20 worker nodes in Kubernetes sending 20 million Analyzed Spans
...
+A [host][3] is a physical or virtual operating system instance. Datadog records the number of hosts you are concurrently monitoring in the Datadog Infrastructure service once an hour. For billing APM, number of hosts with [APM installed][8] and sending traces are calculated every hour. At the end of the month, you are billed based on your 99th percentile usage for [APM hosts][4].
 
 **2. How is billing calculated if I deploy one agent per container?**
 
-It is recommended to [setup running][8] one agent per underlying host for container deployment. If you still choose to run one agent per container, then each container is treated as a single host. The price is then (Price Per APM host) * (No. of containers)
+It is recommended to [setup running][9] one agent per underlying host for container deployment. If you still choose to run one agent per container, then each container is treated as a single host. The price is then (Price Per APM host) * (No. of containers)
```
Comment (sarina-dd) [inline content/en/account_management/billing/apm_distributed_tracing.md]:
```suggestion
It is recommended to [setup running][9] one agent per underlying host for container deployment. If you still choose to run one agent per container, then each container is treated as a single host. The price is then (Price Per APM host) * (Number of containers)
```

File: content/en/account_management/billing/_index.md [hunk 2]
```diff
@@ -21,7 +21,11 @@ It is recommended that containers are monitored with a single containerized Agen
 
 ### Serverless
 
-Datadog bills based on the average number of functions per hour across the month for your accounts. Pro and Enterprise plans include 40 custom metrics per billed function. Every hour, Datadog records the number of functions that were executed one or more times and monitored by your Datadog account. At the end of the month, Datadog charges by calculating the average of the hourly number of functions recorded. For more information, see the [Serverless billing page][4] and the [Datadog Pricing page][5].
+Datadog bills based on the average number of functions per hour across the month for your accounts. Every hour, Datadog records the number of functions that were executed one or more times and monitored by your Datadog account. At the end of the month, Datadog charges by calculating the average of the hourly number of functions recorded. 
+
+Pro and Enterprise plans include 40 custom metrics per billed function. Distributed traces incur charges based on the analyzed spans consumed.
```
Comment (pinkerton) [inline content/en/account_management/billing/_index.md]:
```suggestion
Pro and Enterprise plans include 40 custom metrics per billed function. Billing for serverless APM depends on the total number of Analyzed Spans submitted to the Datadog APM service at the end of the month. There are no billable APM Hosts when using serverless. 
```

Comment (pinkerton) [reply content/en/account_management/billing/_index.md]: let's also link the proper billing nouns (APM Host, Analyzed Span) individually to this page: https://docs.datadoghq.com/account_management/billing/pricing/#apm

Comment (pinkerton) [reply content/en/account_management/billing/_index.md]: that follows the style we use on this page

**Pull request merged**